src/Symfony/Bundle/FrameworkBundle/Command/UidGenerateCommand.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Uid\AbstractUid;
+use Symfony\Component\Uid\Ulid;
+use Symfony\Component\Uid\Uuid;
+
+class UidGenerateCommand extends Command
+{
+    protected static $defaultName = 'uid:generate';
+
+    private const UUID_1 = '1';
+    private const UUID_3 = '3';
+    private const UUID_4 = '4';
+    private const UUID_5 = '5';
+    private const UUID_6 = '6';
+    private const ULID = 'ulid';
+
+    private static $types = [
+        self::UUID_1,
+        self::UUID_3,
+        self::UUID_4,
+        self::UUID_5,
+        self::UUID_6,
+        self::ULID,
+    ];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $typesAsString = implode(', ', self::$types);
+
+        $this
+            ->setDefinition([
+                new InputArgument('type', InputArgument::OPTIONAL, 'The type/version of the generated UID.', null),
+                new InputArgument('namespace', InputArgument::OPTIONAL, 'Namespace for UUID V3 and V5 versions.', null),
+                new InputArgument('name', InputArgument::OPTIONAL, 'Name for UUID V3 and V5 versions.', null),
+            ])
+            ->setDescription('Generates a UID, that can be either a ULID or a UUID in a given version.')
+            ->setHelp(<<<EOF
+The <info>%command.name%</info> generates UID. This can be a ULID or a UUID
+in a given version. Available types are $typesAsString.
+Examples:
+
+  <info>php %command.full_name% ulid</info> for generating a ULID.
+  <info>php %command.full_name% 1</info> for generating a UUID in version 1.
+  <info>php %command.full_name% 3 9b7541de-6f87-11ea-ab3c-9da9a81562fc foo</info> for generating a UUID in version 3.
+
+EOF
+            )
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \LogicException
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (!class_exists(AbstractUid::class)) {
+            throw new \RuntimeException('Unable to execute this command as the Symfony Uid Component is not installed.');
```

Comment:
You should remove the service in the FrameworkExtension file when the `AbstractUid` doesn't exist instead of throwing an exception